middlewares/middlewares.go

```diff
@@ -147,7 +148,41 @@ func JwtMiddleware(key string, algo string) negroni.Handler {
 			return
 		}
 		out := auth.Claims{}
-		if err := tok.Claims([]byte(key), &out); err != nil {
+		var rawkey interface{}
+
+		//Tester l'ensemble des types de clés...
+		if JWKSet == "" {
+			rawkey = []byte(key)
+		} else {
```

Comment:
could you invert this logic?
```go
// start setting this as default
rawkey = []byte(key)
if JWKSet != "" {
...
}

```

Reply:
Done in latest commit.
https://github.com/prest/prest/pull/871/commits/fee30e7ae1162e49e1d88f7fa384c73b016fdc95